hyundai_kia_connect_api/Vehicle.py

```diff
@@ -143,6 +151,24 @@ def last_service_distance(self, value):
         self._last_service_distance_unit = value[1]
         self._last_service_distance = value[0]
 
+    @property
```

Comment:
This gives a location not defined error. Do I just not list it as a setter? 

```
    @location.setter
    def location(self, latitude, longitude):
        self._location_latitude = latitude
        self._location_longitude = longitude
        self._location_last_set_time = datetime.datetime.now(pytz)
```

Reply:
Just updated this PR with one option for you to look at.

Reply:
looks good, perfect

Reply:
> This gives a location not defined error. Do I just not list it as a setter? 
> 
> ```
>     @location.setter
>     def location(self, latitude, longitude):
>         self._location_latitude = latitude
>         self._location_longitude = longitude
>         self._location_last_set_time = datetime.datetime.now(pytz)
> ```

this should be
def location(self, value):
    self._location_latitude = value[0]
    self._location_longtitude = value[1]
    self._location_last_set_time = value[2]


in EU, vehicle cache data has a different time stamp value than location data, so we can keep actual values rather than now. We can decide this in API level